angular/app/styles/menu.scss

```diff
@@ -0,0 +1,351 @@
+/*! HTML5 Boilerplate v4.3.0 | MIT License | http://h5bp.com/ */
```

Comment:
So maybe not copying the whole H5BP stylesheet, since Bootstrap includes `normalize.css` _and_ H5BP already.